src/main/java/fi/pohina/vinkkilista/AppConfig.java

```diff
@@ -20,7 +24,11 @@ public String getGithubClientSecret() {
         return githubClientSecret;
     }
 
-    public String getStage() {
-        return stage;
+    public boolean isProduction() {
+        return "production".equals(stage);
     }
+
+    // public String getStage() {
+    //    return stage;
+    //}
```

Comment:
Do we need to retain this commented method if we do not intend to use it anymore?